fala/apps/adviser/views.py

```diff
@@ -43,33 +42,84 @@ class AccessibilityView(TemplateView):
 class SearchView(ListView):
     def get(self, request, *args, **kwargs):
         self._form = AdviserSearchForm(data=request.GET or None)
-        self._data = self._form.search()
+        if self._form.is_valid():
+            self._region = self._form.region
```

Comment:
I don't remember making so much stuff like this private and it looks strange to my eye, but I could well be wrong. Worth looking at some examples just to check which seems to be more idiomatic.